plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityKnockback.java

```diff
@@ -30,7 +36,7 @@ public EntityKnockback(EntityTag entity) {
 
     @Override
     public String getPropertyString() {
-        return String.valueOf(getAbstractArrow().getKnockbackStrength());
+        return String.valueOf(NMSHandler.getVersion().isAtLeast(NMSVersion.v1_21) ? null : getAbstractArrow().getKnockbackStrength());

```

Comment:
You're passing `null` into `String#valueOf`, which I believe would return `"null"`, I.e. a string? you want to put the version check outside the `valueOf`, so that it actually returns null.